City-of-Helsinki/haitaton-backend
# Hai 233 add organisation api

Added simple organisation api that fetches organisations from db. 

To test
- delete your haitaton-data folder
- recreate the db container
` docker-compose --env-file .env.local up db` 
- Run application (runs liquidbase migrations) from Idea or with 
` ./gradlew bootRun
`
- Check that database includes table organisations with corresponding data
- Make an api request to fetch organisations 
```
$ curl http://localhost:8080/organisaatiot                                                                                                                                         ✔ 
[{"id":1,"tunnus":"DNA","nimi":"DNA Oyj"},{"id":2,"tunnus":"WELHO","nimi":"DNA Welho Oy"},{"id":3,"tunnus":"ELISA","nimi":"Elisa Oyj"},{"id":4,"tunnus":"FREMANTLE","nimi":"FremantleMedia Finland Oy"},{"id":5,"tunnus":"HELEN","nimi":"Helen Oy"},{"id":6,"tunnus":"HELENSAH","nimi":"Helen sähköverkko Oy"},{"id":7,"tunnus":"HELKYMP","nimi":"Helsingin kaupunki KYMP"},{"id":8,"tunnus":"HELTA","nimi":"Helsingin tapahtumasäätiö sr"},{"id":9,"tunnus":"HSY","nimi":"Helsingin seudun ympäristöpalvelut -kuntayhtymä HSY"},{"id":10,"tunnus":"NIEMI","nimi":"Niemi palvelut Oy"},{"id":11,"tunnus":"JOKERI","nimi":"Raide-Jokeri Allianssi / KYMP/MAKA/LIKE"},{"id":12,"tunnus":"SUOILO","nimi":"Suomen Ilotulitus Oy"},{"id":13,"tunnus":"TELIA","nimi":"Telia Finland Oyj"},{"id":14,"tunnus":"VARMATIE","nimi":"Varmatie Oy"},{"id":15,"tunnus":"VIKTOREK","nimi":"Viktor Ek Muutot Oy"},{"id":16,"tunnus":"YELLOW","nimi":"Yellow Film & TV Oy"},{"id":17,"tunnus":"YITSUOMI","nimi":"YIT Suomi Oy"},{"id":18,"tunnus":"YLE","nimi":"Yleisradio Oy"},{"id":19,"tunnus":"ZODIAK","nimi":"Zodiak Finland Oy"}]%
```
